core/src/main/scala/org/broadinstitute/dsde/rawls/workspace/MultiCloudWorkspaceService.scala

```diff
@@ -497,13 +498,28 @@ class MultiCloudWorkspaceService(override val ctx: RawlsRequestContext,
       throw new RawlsExceptionWithErrorReport(ErrorReport(StatusCodes.NotImplemented, "MC workspaces are not enabled"))
     }
 
+    assertBillingProfileCreationDate(profile)
+
     traceWithParent("createMultiCloudWorkspace", parentContext)(s1 =>
       createWorkspace(workspaceRequest, profile, s1) andThen { case Success(_) =>
         createdMultiCloudWorkspaceCounter.inc()
       }
     )
   }
 
+  def assertBillingProfileCreationDate(profile: ProfileModel): Unit = {
+    val previewDate = new DateTime(2023, 9, 12, 0, 0, DateTimeZone.UTC)
+    val isTestProfile = profile.getCreatedDate() == null || profile.getCreatedDate() == ""
```

Comment:
I didn't want to have to update all of our mocked billing profiles to include a createdDate.